### PR TITLE
Patch vul

### PR DESCRIPTION
**Pull request by stefanab**
Run `yarn upgrade` and update snapshot tests. Tests were also failing in `master` branch so is not introduced with these commits

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -65,5 +72,9 @@
     "react-test-renderer": "^16.13.1",
     "tern": "^0.24.3"
   },
-  "dependencies": {}
+  "dependencies": {
+    "node-fetch": "^2.6.1",
+    "plist": ">=3.0.2",
+    "ua-parser-js": "^0.7.24"
```
Comment (Marvedog) [inline package.json]: Not sure why this is needed anymore. Why not just delete it both as a a `dependency` and as a `resolution`?

File: package.json [hunk 2]
```diff
@@ -65,5 +72,9 @@
     "react-test-renderer": "^16.13.1",
     "tern": "^0.24.3"
   },
-  "dependencies": {}
+  "dependencies": {
+    "node-fetch": "^2.6.1",
+    "plist": ">=3.0.2",
```
Comment (Marvedog) [inline package.json]: It seems to me like you can remove this just fine as well. There is no conflicting entries in `yarn.lock`.

File: package.json [hunk 3]
```diff
@@ -65,5 +72,9 @@
     "react-test-renderer": "^16.13.1",
     "tern": "^0.24.3"
   },
-  "dependencies": {}
+  "dependencies": {
+    "node-fetch": "^2.6.1",
```
Comment (Marvedog) [inline package.json]: Why is this added as a dependency when it is already in the `resolutions` list?

**Review by Marvedog — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (stefanab) [reply package.json]: idk. all this is automatic stuff from yarn

Comment (Marvedog) [reply package.json]: If you inspect the `yarn.lock` file, you can see that my statement is correct and that they can be deleted

Comment (Marvedog) [reply package.json]: You can inspect `yarn.lock` to see if they are still needed

Comment (stefanab) [reply package.json]: not sure i understand. isn't the `yarn.lock` file generated based on the contents of `package.json` looks like the ua-parser is used by `fbjs`?
